src/Symfony/Component/Security/Http/Attribute/IsGranted.php

```diff
@@ -14,38 +14,27 @@
 use Symfony\Component\ExpressionLanguage\Expression;
 
 /**
+ * Checks if user has permission to access to some resource using security roles and voters.
+ *
+ * @see https://symfony.com/doc/current/security.html#roles
+ *
  * @author Ryan Weaver <ryan@knpuniversity.com>
  */
 #[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
 final class IsGranted
 {
+    /**
+     * @param string|Expression            $attribute     Sets the first argument that will be passed to isGranted()
+     * @param array|string|Expression|null $subject       Sets the second argument passed to isGranted()
```

Comment:
Maybe we can make these two lines more self explanatory?

Reply:
Tried something else that may be more explicit 👍